app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+###
+#  User class for authenticating via WebAuth
+###
+class User
+  attr_accessor :webauth
+
+  def initialize(webauth)
+    @webauth = webauth
+  end
+
+  def webauth_user?
+    @webauth.present?
```

Comment:
If I don't use `@webauth` then webauth ends up being `nil`. I can't seem to get it to work any other way...